modules/swagger-codegen-common-feign/pom.xml

```diff
@@ -0,0 +1,188 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This is the `pom.xml` for the new module that has the common interface that all feign-based Java codegen clients can use. There are probably some plugins here I can delete. I'll improve this and take any suggestions.